tests/test_ble_connection_discovery_client_targets.py

```
@@ -2,9 +2,10 @@
 
 from __future__ import annotations
 
+import contextlib
 from threading import Event, RLock
 from types import SimpleNamespace
-from typing import Any
+from typing import Any, Iterator
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)